diff --git a/authentik/admin/files/backends/file.py b/authentik/admin/files/backends/file.py
index 6d60d5f23..720223231 100644
--- a/authentik/admin/files/backends/file.py
+++ b/authentik/admin/files/backends/file.py
@@ -52,7 +52,6 @@ class FileBackend(ManageableBackend):
         # file creation on fresh installs.
         return (
             self._base_dir.exists()
-            and (self._base_dir.is_mount() or (self._base_dir / self.usage.value).is_mount())
             or (settings.DEBUG or settings.TEST)
         )
 
